docs-devsite/firestore_.persistentcacheindexmanager.md

```diff
@@ -0,0 +1,37 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# PersistentCacheIndexManager class
```

Comment:
Edit (I don't know if these are generated files, so edits to source may make these .md edits unnecessary.

"A `PersistentCacheIndexManager` for configuring persistent cache indexes used for local query execution."